### ISSUE TITLE
Ios build fails with  symbol(s) not found for architecture x86_64 while integrating flipper

### ISSUE THREAD

**Issue by ashishkumar1000** [facebook/flipper#1178]

**Comment by cekkaewnumchai** [comment]
I believe Flipper is integrated out of the box if you have RN 0.62, but I may misunderstand something

cc. @mweststrate

**Comment by mweststrate** [comment]
Correct, you should need to install anything. If you are upgrading a
project make sure to use the react native upgrade helper to get the correct
configs

**Comment by Natkarock** [comment]
> I am trying to integrate flipper with my react native app. Although the integration is successful for android the build but it fails for ios with below error. I am using react native 0.62.2
> 
> Undefined symbols for architecture x86_64:
> "associated type descriptor for Swift.ExpressibleByFloatLiteral.FloatLiteralType", referenced from:
> protocol conformance descriptor for __C.YGValue : Swift.ExpressibleByFloatLiteral in YogaKit in libYogaKit.a(YGLayoutExtensions.o)
> "associated type descriptor for Swift.ExpressibleByIntegerLiteral.IntegerLiteralType", referenced from:
> protocol conformance descriptor for __C.YGValue : Swift.ExpressibleByIntegerLiteral in YogaKit in libYogaKit.a(YGLayoutExtensions.o)
> "value witness table for Builtin.Int32", referenced from:
> full type metadata for __C.YGUnit in libYogaKit.a(YGLayoutExtensions.o)
> "protocol witness table for Swift.Float : Swift._ExpressibleByBuiltinFloatLiteral in Swift", referenced from:
> associated type witness table accessor for Swift.ExpressibleByFloatLiteral.FloatLiteralType : Swift._ExpressibleByBuiltinFloatLiteral in __C.YGValue : Swift.ExpressibleByFloatLiteral in YogaKit in libYogaKit.a(YGLayoutExtensions.o)
> "Swift.Float.init(Swift.Double) -> Swift.Float", referenced from:
> static (extension in YogaKit):CoreGraphics.CGFloat.% postfix(CoreGraphics.CGFloat) -> __C.YGValue in libYogaKit.a(YGLayoutExtensions.o)
> (extension in YogaKit):__C.YGValue.init(CoreGraphics.CGFloat) -> __C.YGValue in libYogaKit.a(YGLayoutExtensions.o)
> "protocol witness table for Swift.Int : Swift._ExpressibleByBuiltinIntegerLiteral in Swift", referenced from:
> associated type witness table accessor for Swift.ExpressibleByIntegerLiteral.IntegerLiteralType : Swift._ExpressibleByBuiltinIntegerLiteral in __C.YGValue : Swift.ExpressibleByIntegerLiteral in YogaKit in libYogaKit.a(YGLayoutExtensions.o)
> "protocol descriptor for Swift.ExpressibleByFloatLiteral", referenced from:
> protocol conformance descriptor for __C.YGValue : Swift.ExpressibleByFloatLiteral in YogaKit in libYogaKit.a(YGLayoutExtensions.o)
> "method descriptor for Swift.ExpressibleByFloatLiteral.init(floatLiteral: A.FloatLiteralType) -> A", referenced from:
> protocol conformance descriptor for __C.YGValue : Swift.ExpressibleByFloatLiteral in YogaKit in libYogaKit.a(YGLayoutExtensions.o)
> "associated conformance descriptor for Swift.ExpressibleByFloatLiteral.Swift.ExpressibleByFloatLiteral.FloatLiteralType: Swift._ExpressibleByBuiltinFloatLiteral", referenced from:
> protocol conformance descriptor for __C.YGValue : Swift.ExpressibleByFloatLiteral in YogaKit in libYogaKit.a(YGLayoutExtensions.o)
> "protocol descriptor for Swift.ExpressibleByIntegerLiteral", referenced from:
> protocol conformance descriptor for __C.YGValue : Swift.ExpressibleByIntegerLiteral in YogaKit in libYogaKit.a(YGLayoutExtensions.o)
> "method descriptor for Swift.ExpressibleByIntegerLiteral.init(integerLiteral: A.IntegerLiteralType) -> A", referenced from:
> protocol conformance descriptor for __C.YGValue : Swift.ExpressibleByIntegerLiteral in YogaKit in libYogaKit.a(YGLayoutExtensions.o)
> "associated conformance descriptor for Swift.ExpressibleByIntegerLiteral.Swift.ExpressibleByIntegerLiteral.IntegerLiteralType: Swift._ExpressibleByBuiltinIntegerLiteral", referenced from:
> protocol conformance descriptor for __C.YGValue : Swift.ExpressibleByIntegerLiteral in YogaKit in libYogaKit.a(YGLayoutExtensions.o)
> "__swift_FORCE_LOAD_$_swiftCompatibility50", referenced from:
> __swift_FORCE_LOAD_$_swiftCompatibility50_$_YogaKit in libYogaKit.a(YGLayoutExtensions.o)
> (maybe you meant: __swift_FORCE_LOAD_$_swiftCompatibility50_$_YogaKit)
> "__swift_FORCE_LOAD_$_swiftCompatibilityDynamicReplacements", referenced from:
> __swift_FORCE_LOAD_$_swiftCompatibilityDynamicReplacements_$_YogaKit in libYogaKit.a(YGLayoutExtensions.o)
> (maybe you meant: __swift_FORCE_LOAD_$_swiftCompatibilityDynamicReplacements_$_YogaKit)
> "_swift_getForeignTypeMetadata", referenced from:
> type metadata accessor for __C.YGValue in libYogaKit.a(YGLayoutExtensions.o)
> type metadata accessor for __C.YGUnit in libYogaKit.a(YGLayoutExtensions.o)
> ld: symbol(s) not found for architecture x86_64
> clang: error: linker command failed with exit code 1 (use -v to see invocation)

hello! did you find a solution?

**Comment by mweststrate** [comment]
For future readers: please don't comment on closed issues as they offer limited exposure. Rather open new issues detailing dependency versions and build setup after checking our troubleshooting guide: https://fbflipper.com/docs/troubleshooting/#react-native

**Comment by kumbhar-ketan** [comment]
Hey Me too getting this issue. Can you please help us!

**Comment by nathanhleung** [comment]
Following the instructions in this thread fixed the issue for me: https://developer.apple.com/forums/thread/655438
